app/generate.js

```diff
@@ -16,6 +16,15 @@ function importAssets(name, theme) {
   }
 }
 
+function generateHeader(theme, headerUrl, headerDescription, headerTagline) {
```

Comment:
It's a logoUrl and a logoDescription. They don't link to and describe the entire header.

Reply:
Also, you sure have a lot of arguments here...

Reply:
Check updated pullrequest